frontend-service/src/app/js/App.js

```diff
@@ -1,25 +1,19 @@
 import React from "react";
-import logo from "../../assets/logo.svg";
 import "../css/App.css";
+import { Provider} from 'react-redux'; 
+import configureStore from './store/configureStore';
+import {Route, Switch} from 'react-router-dom';
+import Landing from "./components/Landing";
+
+const store = configureStore(); 
 
 function App() {
   return (
-    <div className="App">
```

Comment:
Create a new markdown file, titled `REACT-COMPONENTS.md` . In it, specify the components you have created and what they do so that anyone who would be working on this later, would know the components you have already created and the states and props you're currently using.

Reply:
Alright